package.json

```diff
@@ -1,24 +1,25 @@
 {
-  "name": "xr-input",
-  "version": "1.0.0",
+  "name": "action-input",
+  "version": "0.1.0",
   "description": "A framework-agnostic input library that progressively handles flat, portal, and immersive web apps.",
```

Comment:
The same as on the description, removing the  `flat, portal and immersive` as is not part of the library but part of the framework that should use it